packages/mui-material/src/Typography/typography.spec.tsx

```diff
@@ -32,7 +32,8 @@ const typographyTest = () => {
       <Typography align="right" color="primary" display="initial" />
       <Typography align="justify" color="secondary" display="initial" />
       <Typography align="inherit" color="text.primary" />
```

Comment:
```suggestion
      <Typography align="inherit" color="textPrimary" />
```

Reply:
```js
<Typography align="inherit" color="text.primary" />
```

This is intentional to test that `text.primary` type still works.